core/src/test/resources/classloaderjson/META-INF/openapi.json

```diff
@@ -0,0 +1,9 @@
+{
+	"openapi": "3.1.0",
+	"info": {
+		"title": "Loaded from the class path",
```

Comment:
Should change this line so that it's not identical to the yaml file from the other test.